### DISCUSSION TITLE
Freeze upon launch

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21658] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

Not exactly a crash, but pretty close; 

osu will freeze upon launch and do this rendering bug. happens about 2/3rds of the time, and happens in my dev build on latest code. 

### Screenshots or videos

![image](https://user-images.githubusercontent.com/78441708/208043095-f794c761-c91c-400f-a2cf-e0dff2e4fa8b.png)

### Version

2022.1208.0 & latest dev build

### Logs

latest crash, these first 2 were updated and the last 3 were not, however the last 3 were updated on a seemingly identical crash.
[network.log](https://github.com/ppy/osu/files/10243443/network.log)
[runtime.log](https://github.com/ppy/osu/files/10243444/runtime.log)

[database.log](https://github.com/ppy/osu/files/10243445/database.log)
[performance.log](https://github.com/ppy/osu/files/10243446/performance.log)
[input.log](https://github.com/ppy/osu/files/10243447/input.log)

**Comment by peppy** [comment, marked as answer]
May be fixed with next framework bump.

**Reply by Axilotl17** [reply]
appears to be so.
